### DISCUSSION TITLE
Searching by tags

### DISCUSSION THREAD

**Discussion by ItsNoted** [diced/zipline#282] [Ideas]
I'd like to suggest a search feature so we can somehow search for images or pastes we want to quickly reference. Maybe allow tagging an upload then search by tag?

**Comment by diced** [comment]
Good idea, I think I'll consider it for 3.7.1.

**Comment by tigattack** [comment]
This would be really great. I'd love to migrate to Zipline from XBackBone, after a quick trial it's nicer to use in almost every way, but being unable to search by filename and/or tags is really a killer for me 😕

**Comment by diced** [comment]
should've been closed a while ago but it is supported in v4.
